Sofa/framework/Simulation/Core/src/sofa/simulation/Simulation.h

```diff
@@ -32,93 +31,153 @@ namespace sofa::simulation
     typedef sofa::core::sptr<Node> NodeSPtr;
 }
 
-namespace sofa
+namespace sofa::simulation
 {
 
-namespace simulation
-{
+/// Initialize the objects
+void SOFA_SIMULATION_CORE_API initNode(Node* root);
+///Init a node without changing the context of the simulation.
+void SOFA_SIMULATION_CORE_API initNodeNoContextUpdate(Node* node);
```

Comment:
Simulation::init() renamed/moved to InitNode()
Simulation::InitNode() renamed/moved to initNodeNoContextUpdate()

This is a bit confusing I think.
I guess Simulation::init() could not moved to a free function called simulation::init(), but what about initRoot(...) ?

Reply:
Sorry, I did not understand your suggestion